Demo/Application/Features/PayPal - Native Checkout/BraintreeDemoPayPalNativeCheckoutViewController.swift

```diff
@@ -5,36 +5,46 @@ import BraintreePayPalNativeCheckout
 class BraintreeDemoPayPalNativeCheckoutViewController: BraintreeDemoPaymentButtonBaseViewController {
     lazy var payPalNativeCheckoutClient = BTPayPalNativeCheckoutClient(apiClient: apiClient)
 
+    func checkoutPaymentButton(title: String) -> UIButton {
+      let button = UIButton(type: .system)
+      button.setTitle(title, for: .normal)
+      button.setTitleColor(.blue, for: .normal)
+      button.setTitleColor(.lightGray, for: .highlighted)
+      button.setTitleColor(.lightGray, for: .disabled)
+      button.translatesAutoresizingMaskIntoConstraints = false
+      return button
+    }
+
     override func createPaymentButton() -> UIView! {
-        let payPalCheckoutButton = UIButton(type: .system)
-        payPalCheckoutButton.setTitle("One Time Checkout", for: .normal)
-        payPalCheckoutButton.setTitleColor(.blue, for: .normal)
-        payPalCheckoutButton.setTitleColor(.lightGray, for: .highlighted)
-        payPalCheckoutButton.setTitleColor(.lightGray, for: .disabled)
+        let payPalCheckoutButton = checkoutPaymentButton(title: "One Time Checkout")
         payPalCheckoutButton.addTarget(self, action: #selector(tappedPayPalCheckout), for: .touchUpInside)
-        payPalCheckoutButton.translatesAutoresizingMaskIntoConstraints = false
 
-        let payPalVaultButton = UIButton(type: .system)
-        payPalVaultButton.setTitle("Vault Checkout", for: .normal)
-        payPalVaultButton.setTitleColor(.blue, for: .normal)
-        payPalVaultButton.setTitleColor(.lightGray, for: .highlighted)
-        payPalVaultButton.setTitleColor(.lightGray, for: .disabled)
-        payPalVaultButton.addTarget(self, action: #selector(tappedPayPalVault), for: .touchUpInside)
-        payPalVaultButton.translatesAutoresizingMaskIntoConstraints = false
+        // Buyers are shown a billing agreement without purchase
+        // For more information: https://developer.paypal.com/braintree/docs/guides/paypal/vault/javascript/v3/
+        let vaultCheckoutButton = checkoutPaymentButton(title: "Vault Checkout")
+        vaultCheckoutButton.addTarget(self, action: #selector(tappedVaultCheckout), for: .touchUpInside)
```

Comment:
Take it or leave it, but since we extracted the rest of the button into a method thoughts on making this change as well to move `addTarget` into the helper method as well? Something like:
```swift
func checkoutPaymentButton(title: String, action: Selector) -> UIButton {
    ...
    button.addTarget(self, action: action, for: .touchUpInside)
    return button
}

// callsite
let payPalCheckoutButton = checkoutPaymentButton(title: "One Time Checkout", action: #selector(tappedPayPalCheckout))
```

Reply:
Thanks for the suggestion!